parsons/google/google_bigquery.py

```diff
@@ -464,7 +467,7 @@ def copy_from_gcs(
                     job_config=job_config,
                     **load_kwargs,
                 )
-                load_job.result()
+                load_job.result(timeout=MAX_TIMEOUT)
```

Comment:
Instead of using a constant, perhaps this should be a keyword parameter that defaults to 30. Something like:

`def get_table_ref(client, table_name, timeout = 30):`

That way if 30 isn't enough for my use case, or is too long, I can adapt the behavior.

Reply:
Hi, for sure! I think that is a cleaner implementation, good suggestion